src/groupeddataframe/splitapplycombine.jl

```diff
@@ -266,7 +266,7 @@ Last Group: 1 row
 
 See [`by`](@ref) for more examples.
 """
-function Base.map(f::Any, gd::GroupedDataFrame)
+function Base.map(f::Function, gd::GroupedDataFrame)
```

Comment:
I would drop it.

Not dropping it would need a messy code (we do not support passing `NamedTuple` of transformations now anywhere else).

The reason I disallowed it is that now we are inconsistent with `map` contract (first argument must be a function in Base). Also, although it is slower, you can do `map` with `SubDataFrame` if you want (in general I think `map` is hardly used now anyway).

Finally, as you have noted we will have `select` for this soon.

Reply:
Fine with me.